### PR TITLE
Update TorBrowser to v10.5

### PR DESCRIPTION
**Pull request by Trenly**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
```raw
PS D:\Git\winget-pkgs> winget validate --manifest D:\Git\winget-pkgs\manifests\t\TorProject\TorBrowser\10.5.0
Manifest validation succeeded.

PS D:\Git\winget-pkgs> winget install -m D:\Git\winget-pkgs\manifests\t\TorProject\TorBrowser\10.5.0
Found Tor Browser [TorProject.TorBrowser]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://www.torproject.org/dist/torbrowser/10.5/torbrowser-install-win64-10.5_en-US.exe
  ██████████████████████████████  70.6 MB / 70.6 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20051)

### CODE REVIEW

**Comment by Trenly** [comment]
@ImJoakim Would you be able to help me understand why this might be failing? Both packages install perfectly fine on my local machine, and I'm not seeing anything in the pipeline outputs indicating why it failed the installation validation

**Comment by ImJoakim** [comment]
> @ImJoakim Would you be able to help me understand why this might be failing? Both packages install perfectly fine on my local machine, and I'm not seeing anything in the pipeline outputs indicating why it failed the installation validation

@Trenly Try closing the PR and reopening it again, this will make it run through the pipeline again.

File: manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TorProject.TorBrowser
+PackageVersion: 10.5.0
```
Comment (ItzLevvie) [inline manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 10.5
```

File: manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TorProject.TorBrowser
+PackageVersion: 10.5.0
```
Comment (ItzLevvie) [inline manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.yaml:5]:
```suggestion
PackageVersion: 10.5
```

File: manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TorProject.TorBrowser
+PackageVersion: 10.5.0
```
Comment (ItzLevvie) [inline manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.installer.yaml:5]:
```suggestion
PackageVersion: 10.5
```

Comment (ItzLevvie) [reply manifests/t/TorProject/TorBrowser/10.5.0/TorProject.TorBrowser.installer.yaml:5]: image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**